### PR TITLE
proxy

### PR DESCRIPTION
**Pull request by Hervey424**

### CODE REVIEW

**Comment by hyeonsangjeon** [comment]
Is there any faults with proxy code?
Looking at the current code, it seems that the server process assigns a value to the proxy variable when it first creates a thread.
There are no comments in the pull request, so it is difficult to understand. Leave a comment.

**Comment by Hervey424** [comment]
No matter how you set the proxy, the proxy is always empty in the youtube-dl command line

**Comment by hyeonsangjeon** [comment]
Sorry for the late reply. Did you test it even when the proxy is not used? I'll merge it tomorrow.

File: youtube-dl-server.py [hunk 1]
```diff
@@ -123,18 +123,20 @@ def dl_worker():
...
-    elif (url[2] == "audio-m4a" or url[2] == "audio"):
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestaudio[ext=m4a]", "--exec", "touch {} && mv {} ./downfolder/", url[0]]
-    elif (url[2] == "audio-mp3"):
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestaudio[ext=m4a]", "-x", "--audio-format", "mp3", "--exec", "touch {} && mv {} ./downfolder/", url[0]]
-    else:
-        resolution = url[2][:-1]
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestvideo[height<="+resolution+"][ext=mp4]+bestaudio[ext=m4a]", "--exec", "touch {} && mv {} ./downfolder/",  url[0]]
-    print (" ".join(cmd))
-    return cmd
+    with open('Auth.json') as data_file:
```
Comment (tlvu) [inline youtube-dl-server.py:126]: @Hervey424 Could you tell me the location of this `Auth.json` file on disk, its expected content, and who or which code is supposed to create it?

Comment (tlvu) [reply youtube-dl-server.py:126]: Opps, sorry, found it here https://github.com/hyeonsangjeon/youtube-dl-nas/blob/1c3318a259879c2e709c70ee0d7a0368fa4dcfd6/Auth.json

**Pull request merged**